backend-rust/src/indexer.rs

```diff
@@ -918,6 +923,11 @@ impl PreparedBlock {
         )
         .await?;
         let baker_unmark_suspended = PreparedUnmarkPrimedForSuspension::prepare(data)?;
+
+        let election_info =
+            node_client.get_election_info(data.finalized_block_info.height).await?.response;
```

Comment:
While I understand the need for doing this on the block level, I do not think we should do this for every block. This is a heavy operation.

My suggestion would be that we do this not a all while catching up and while caught up then every say 30 block or so (~1m).

Reply:
Changed it to only do the calculation on a payday block (once every day).